content/en/docs/setup/platform-setup/kubesphere/index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: KubeSphere Container Platform
+description: Instructions to setup a KubeSphere Container Platform for Istio.
+weight: 19
+skip_seealso: true
+keywords: [platform-setup,kubesphere,kubernetes]
+---
+
+Follow these instructions to prepare the [KubeSphere Container Platform](https://github.com/kubesphere/kubesphere) for Istio. You can download KubeSphere to easily install a Kubernetes cluster on your Linux machines.
+
+{{< tip >}}
+KubeSphere provides [All-in-One](https://kubesphere.io/docs/v2.1/en/installation/all-in-one/) and [Multi-Node](https://kubesphere.io/docs/v2.1/en/installation/multi-node/) installations. This enables quick setup and manages Kubernetes and Istio in a unified web console. This tutorial will walk you through the All-in-One installation. Reference [Multi-node Installation](https://kubesphere.io/docs/v2.1/en/installation/multi-node/) for further information.
+{{< /tip >}}
+
+## Prerequisites
+
+A new Linux machine, supports virtual machine or bare metal, the minimum requirements are listed:
+
+* Hardware
+  * CPU: at least 2 Cores
```

Comment:
I don't find there is [unordered list indentation](https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md007---unordered-list-indentation) in line 20, but lint check returns error here, can someone help to check?

Reply:
Noted below I changed the indent to 4 spaces on the indented unordered lists.

Reply:
@ericvn I am confused about here since [unordered list indentation](https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md007---unordered-list-indentation) gives an example with 2 spaces... but 4 spaces are actually correct.